src/components/matching/FeedbackTypeCheckbox.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import type { ComponentProps } from 'react';
+
+interface FeedbackTypeCheckboxProps extends ComponentProps<'input'> {}
+
+const FeedbackTypeCheckbox = ({
+  style,
+  children,
+  ...otherProps
+}: FeedbackTypeCheckboxProps) => {
+  return (
+    <label>
+      <input
+        type="checkbox"
+        style={
+          style !== null && style !== undefined
+            ? { ...style, display: 'none' }
+            : { display: 'none' }
+        }
```

Comment:
p1;
style props를 전달하지 않으면 특정 스타일이 적용되지 않기 때문에 다음과 같이 작성해도 우상님이 의도한 것이 동일하게 적용될 것 같습니다.
다만, style props를 자유도 높게 전달할 것인지 같이 논의해보면 좋을 것 같습니다.

```suggestion
        style={{ 
          ...style, 
          display: 'none', 
        }}
```

Reply:
!!그렇네요 감사합니다~! 해당 부분 수정하겠습니다 :)